Runtime/Settings/Settings.cs

```diff
@@ -135,15 +146,73 @@ protected void Set<S>( ref S setting, S value )
         }
 
         // Marks the settings dirty so that it will be saved.
-        protected new void SetDirty() => EditorUtility.SetDirty( this );
+        protected new void SetDirty()
+        {
+#if UNITY_EDITOR
+            EditorUtility.SetDirty( this );
+#endif
+            _isDirty = true;
+        }
 
         // The directory name of the current project folder.
         static string GetProjectFolderName()
         {
             var path = Application.dataPath.Split( '/' );
             return path[ path.Length - 2 ];
         }
-#endif
+
+        /// <summary>
+        /// Saves the current setting to disk, only for RuntimeUser. Skips it if the setting is not dirty (not set recently)
+        /// </summary>
+        /// <param name="ignoreDirtyFlag">Set this true to ignore the dirty flag check</param>
+        public static void SaveSettings(bool ignoreDirtyFlag = false)
+        {
+            if( attribute.usage != SettingsUsage.RuntimeUser )
+            {
+                Debug.LogWarning( "Save method is only used with RuntimeUser settings" );
+                return;
+            }
+
+            if( !ignoreDirtyFlag && !_isDirty )
+            {
+                Debug.Log( "Settings are clean, no save needed" );
+                return;
+            }
+
+            if( !Directory.Exists( GetSettingsPath() ) )
+                Directory.CreateDirectory( GetSettingsPath() );
+
+            var filepath = GetSettingsPath() + ( attribute.filename ?? typeof(T).Name ) + ".json";
+            File.WriteAllText( filepath, JsonUtility.ToJson( _instance, true ) );
+            _isDirty = false;
+        }
+
+        /// <summary>
+        /// Load the current setting from disk, only for RuntimeUser
+        /// </summary>
+        /// <returns>True if load goes successfully, false if errors are present</returns>
+        public static bool LoadSettings()
```

Comment:
Rename the method to `TryLoadSettings()` (implies bool result, does not throw), rather than `LoadSettings()` (implies throwing on error).